src/crimson/os/seastore/random_block_manager/nvmedevice.h

```diff
@@ -37,28 +65,121 @@ namespace crimson::os::seastore::nvme_device {
  *
  * For more information about NVMe protocol, refer https://nvmexpress.org/
  */
-
 struct nvme_identify_command_t {
   uint32_t common_dw[10];
+
   uint32_t cns : 8;
   uint32_t reserved : 8;
-  uint32_t cntroller_id : 16;
+  uint32_t cnt_id : 16;
 
   static const uint8_t CNS_NAMESPACE = 0x00;
   static const uint8_t CNS_CONTROLLER = 0x01;
 };
 
 struct nvme_admin_command_t {
-  union
-  {
-    nvme_passthru_cmd common_cmd;
-    nvme_identify_command_t identify_cmd;
+  union {
+    nvme_passthru_cmd common;
+    nvme_identify_command_t identify;
   };
 
   static const uint8_t OPCODE_IDENTIFY = 0x06;
 };
 
-using NVMePassThroughCommand = nvme_passthru_cmd;
```

Comment:
A quick one line summary of what each of these structures is used for might help.